public/app/features/logs/components/LogRows.tsx

```diff
@@ -48,6 +49,18 @@ class UnThemedLogRows extends PureComponent<Props, State> {
 
   state: State = {
     renderAll: false,
+    contextIsOpen: false,
+  };
+
+  /**
+   * Toggle the `contextIsOpen` state when a context of one LogRow is opened in order to not show the menu of the other log rows.
+   */
+  toggleAllContexts = (): void => {
```

Comment:
Can I ask why did you decide to name this `toggleAllContexts` as now we can show only 1 log context per all rows. So maybe it could be `toggleContextIsOpen`? 🤔

Reply:
Because I am bad at naming things :D 
Good idea, done in a919576d5

Reply:
Because I am bad at naming things. Done in a919576d5a42a006af02f7335f41592500a6bca4